From 783519beb76afb81c0a20527925d6af70e049471 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 11 Jul 2018 13:18:29 +0300
Subject: [PATCH] board: habana: substitute package list

There is a variable with the package list - use it where appropriate.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/board/habanalabs/common/local_install_packages b/board/habanalabs/common/local_install_packages
index 325c68e..75118a4 100755
--- a/board/habanalabs/common/local_install_packages
+++ b/board/habanalabs/common/local_install_packages
@@ -32,7 +32,7 @@
 
 function local_remove_packages() {
 	local_msg "************* Removing old habana packages *************"
-	sudo apt-get purge --auto-remove habanalabs habanatools libhabana -y
+	sudo apt-get purge --auto-remove $habana_pkg_lst -y
 	sudo apt-get clean
 }
 
